source/_components/light.http_rgb.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: "http_rgb"
+description: "Instructions how to setup http_rgb lights within Home Assistant."
+date: 2017-10-13 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.55
+---
+
+The `http_rgb` light platform lets you control lights that use a simple [HTTP API protocol](git@github.com:robhowlett/http-rgb).
```

Comment:
This should point to the protocol but it goes to the binding repository. And we don't want to clone the repo here.